### ISSUE TITLE
Why gamma=4?

### ISSUE THREAD

**Issue by zhixwang** [cuhk-eda/neural-ilt#3]
Dear authors,

Hi, thanks for the work. I noticed that when you compute the ILT loss, the value of gamma you used is 4, instead of commonly used value of 2:
`ilt_loss = (result - target).pow(4).sum()`

Is there a specific reason that this value is used?

Thanks!

**Comment by InfamousMeGa** [comment]
Dear zhixwang,

We followed the same setting of MOSAIC* (DAC'14). Please refer to section 3.3 of MOSAIC paper for more details.

Gamma can be changed to 2, but may result in different convergence.

\* Gao, Jhih-Rong, Xiaoqing Xu, Bei Yu, and David Z. Pan. "MOSAIC: Mask optimizing solution with process window aware inverse correction." In Proceedings of the 51st Annual Design Automation Conference, pp. 1-6. 2014.

Thanks,
Bentian
